src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -950,7 +950,7 @@ private static string Pluralize(int number)
         {
             using (var client = new RavenAwsGlacierClient(settings, progress, TaskCancelToken.Token))
             {
-                var key = CombinePathAndKey(_database.Name, folderName, fileName);
+                var key = CombinePathAndKey(settings.RemoteFolderName, folderName, fileName);
```

Comment:
I think this should be `settings.RemoteFolderName ?? _database.Name` to maintain backward compatibility

Reply:
done